inventory/host_vars/_example.com/config.yml

```diff
@@ -4,3 +4,9 @@ domain: example.com
 rails_env: production
 
 admin_email: admin@example.com
+
+mail_domain: example.com
+mail_host: smtp.mandrillapp.com
+mail_port: 587
```

Comment:
For security reasons I would move host and port to secrets

Reply:
Done in https://github.com/coopdevs/katuma_secrets/pull/3